Regular Expressions/RegExp.sublime-syntax

```diff
@@ -25,7 +25,7 @@ contexts:
     - include: escaped_char
     - include: charset
     - include: group
-    - match: '([?*+][?+]?)|\{\d*,?\d*\}'
+    - match: '([?*+][?+]?)|\{\d+(,\d*)?\}'
```

Comment:
It looks like `{,9}` may be a quantifier in some languages. Now it goes to wbond's decision.

--

Just tried in ST's console.
```python
>>> re.search(r'a{,9}', 'b') is None
False
```
So `a{,9}` matches `b` in Python 3.3.

--

Meanwhile, for Javascript, the console in Google Chrome,
```javascript
> /a{,9}/.test('b')
false
```
So, `a{,9}` is treated as plain text in Javascript.